src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        String[] resultList = new String[]{};
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
```

Comment:
it's better to use try with resources instead of try catch finally. This way connection to file will be closed automatically when you exit try block